docs/tutorials/local_iteration_series/testing_with_artificial_conditions.md

```diff
@@ -0,0 +1,173 @@
+
+Thankfully there are a number of tools that can simulate adverse network conditions.
+
+For testing locally within the editor we suggest using [Unity Transport - Simulator Tools](#unity-transport---simulator-tools) along with [clone-based workflow via ParrelSync](testing_locally.md#parrelsync).
+
+For testing development builds with built-in artificial latency we suggest using [Unity Transport Simulator Tools with some custom code to inject artifical conditions into the build](#debug-builds).
+
+For testing release builds we suggest using either [Clumsy](#clumsy-on-windows) if you're on Windows, and [Network Link Conditioner](#network-link-conditioner-on-osx) if you're on Mac OS.
+
+## General advice
+
+It's not immediately obvious what the values and options we should enable in our network conditioning tools. All of them allow us to alter various aspects of network conditions such as latency, jitter and packet loss, though the names for concepts and the specific functionality varies between tools.
+
+The values we would use would be different depending on the exact debugging goal, but generally we want our artificial conditions to somewhat match up to what our users would be dealing with in reality.
+
+So how do we reason about what our lag threshold is? 
+To answer this question we can look for information regarding typical and acceptable delays for our game genre (Fighting games, First Person Shooters etc are on one side of the spectrum - these games require fast reaction times and as such have less of an acceptable threshold for lag, whereas the Realtime Strategy games are on the other side of the spectrum, generally being tolerant of much higher delays).
+
+Secondly, to determine how much chaos we want to introduce we can imagine the situations in which people will be playing the game:
+ - Are they playing from their homes (with good internet connections, but a possibility of a WiFi network)?
+ - Are they playing from their phones, using mobile networks of varying quality?
+
+This question tells us if our users are likely to experience more jitter and packet loss - mobile networks are notorious for having widely varying quality of connection.
+
+What's important here is that testing purely with added delay and no packet loss and jitter is extremely unrealistic. These values shouldn't be high - the baseline scenario is not what we would call stress-testing, but they should be non-zero.
+
+Adding jitter to the base delay value adds a layer of chaotic unreliability that would make our peers behave in a more natural way.
+Adding packet loss, apart from introducing even more effective delay to our system could also wreak havoc on our unreliable messages, thus allowing us to explore if we need more defensive logic surrounding our unreliable messages or if we should opt for a reliable message instead.
+
+### Different network conditions for different peers
```

Comment:
```suggestion
Adding packet loss, apart from introducing even more effective delay to our system could also wreak havoc on our unreliable messages, thus allowing us to explore if we need more defensive logic surrounding our unreliable messages or if we should opt for a reliable message instead.

Testing Boss Room on desktop, we've used artificial lag between 80ms and 120ms with 5%-10% packet loss.

### Different network conditions for different peers
```

Reply:
mentioned the numbers we use elsewhere in the text